nx-X11/config/cf/X11.tmpl

```diff
@@ -1154,19 +1143,23 @@ FCHOWN_DEFINES = -DHAS_FCHOWN
 #endif
 
 #ifndef XDamageDefines
-#if BuildDamage
+#if UseInstalledDamageLib
 #define XDamageDefines -DDAMAGE
 #else
 #define XDamageDefines /**/
 #endif
+#else
+#define XDamageDefines /**/
```

Comment:
This looks suspicious. If `XDamageDefines` is not defined and we’re using the external `libXDamage`, set `XDamageDefines` to `-DDAMAGE`. Ok. If there’s no external `libXDamage`, set `XDamageDefines` to empty, so that `XDamage` is not used/supported by `NX` at all. Makes sense.

What **doesn’t make sense** is to **reset** XDamageDefines to empty if it’s **already defined**, though?!